ci/test_external.sh

```diff
@@ -8,8 +8,14 @@ rapids-logger "Create test_external conda environment"
 
 RAPIDS_VERSION=23.10.*
 
-rapids-mamba-retry create -n test_external -c rapidsai -c conda-forge -c nvidia  \
-    cudf=${RAPIDS_VERSION} dask-cudf=${RAPIDS_VERSION} python=3.10 cudatoolkit=11.8
+rapids-mamba-retry create -n test_external -c rapidsai-nightly -c nvidia -c conda-forge \
+    cuxfilter=${RAPIDS_VERSION} cudf=${RAPIDS_VERSION} dask-cudf=${RAPIDS_VERSION} \
+    python=${RAPIDS_PY_VERSION} cuda-version=${RAPIDS_CUDA_VERSION}
```

Comment:
```suggestion
rapids-mamba-retry create \
    -n test_external \
    --override-channels \
    -c rapidsai-nightly \
    -c nvidia \
    -c conda-forge \
    cuxfilter="${RAPIDS_VERSION}" cudf="${RAPIDS_VERSION}" dask-cudf="${RAPIDS_VERSION}" \
    python="${RAPIDS_PY_VERSION}" cuda-version="${RAPIDS_CUDA_VERSION}"
```

Suggesting a few changes here:

- use multiple lines for readability
- use the `--override-channels` flag, otherwise the channels included in the CI image (like the `rapidsai` stable channel) will still be used.
  - docs here - https://docs.conda.io/projects/conda/en/latest/commands/install.html#Channel%20Customization